scripts/call_contracts.py

```diff
@@ -349,7 +355,7 @@ def whitelist(
 @cli.command("update-token")
 @click.argument("token-address", type=str, metavar="ADDRESS", callback=validate_address)
 @click.option("--transfer-limit", type=int, help="New transfer limit of token")
-@click.option("--min-lp-fee", type=int, help="New minimum lp fee")
+@click.option("--rate", type=int, help="ETH/token conversion rate")
```

Comment:
Nitpick: I'd prefer a more explicit option name and help text, e.g.

```python
@click.option("--eth-per-token", type=int, help="ETH per token, in wei")
```

Reply:
wouldnt it be token per eth then?

Reply:
Unfortunately finance languange describes price feeds the other way around (ETH/USD means 1800 USD per ETH)

Reply:
Perfectly illustrates why I am asking this. :-)

Reply:
I don't really care whichever way, but making it more explicit in terms of `--X-per-Y` would be much more obvious, at least for me. I don't really care what X and Y are.

Reply:
or I call it simply `eth-token-rate`?

Reply:
ok I will write `token-per-eth` but writing it with `/` should be `ETH/token`

Reply:
> Unfortunately finance languange describes price rates the other way around (ETH/USD means 1800 USD per ETH)

I have no idea how relevant the source is, but [this](https://saylordotorg.github.io/text_international-finance-theory-and-policy/s07-02-exchange-rate-definitions.html) says:

> It is important to note that the value of a currency is always given in terms of another currency. Thus the value of a U.S. dollar in terms of British pounds is the £/$ exchange rate. The value of the Japanese yen in terms of dollar is the $/¥ exchange rate.

Given that, I would interpret `ETH/USD` as `the value of 1 USD, in terms of ETH`. Which is how I have always been interpreting ratios.

Reply:
> I'm really lacking a financial background so I can't really say what rate is supposed to mean, but from the discussion so far it seems to mean exactly the inverse of what a regular mathematical ration (X / Y) means.

yes exactly, it is the exact opposite of the mathematical ratio.

Reply:
> Alternatively, we could go with just rate as in your original approach, if that is what you'd prefer, but I am afraid we would need to be crystal clear (i.e. by documenting) what it actually means.

This is actually where I think, most people would completely understand the conversion rate as done financial lingo. Or rather I would assume most people could follow this approach. 
Knowing that it is the exact opposite of how it is expressed in mathematical terms. 

To be more precise, I think for `eth-token-rate` most people would understand `how many token are equivalent to 1 ETH`. I also do understand that this is completely counter intuitive for you, if you only have a mathematical background.
I think it's a dilemma here

Reply:
I'm actually leaning towards using the above mentioned rate, so the opposite to the mathematical term, as I think it will be understood by the broader set of people intuitively.
Also if people want to compare the configured rate to the current rate somewhere online, they will find the same value under the ticker `ETH/Token`.

Reply:
Yeah, after some searching online, I think most places interpret `ETH/USD` as `ETH in USD` or `ETH to USD`, so I guess it makes much more sense to interpret the `/` as meaning `in` or `to` in a financial context.

In that light, I'm fine with your suggestion `eth-token-rate`.

(We could also consider e.g. `--eth-to-token-rate` for a tiny bit more explicitness, but I don't mind.)

BTW, I'm curious what you think about my comment [here](https://github.com/beamer-bridge/beamer/pull/1560#discussion_r1147326658).

Reply:
> BTW, I'm curious what you think about my comment [here](https://github.com/beamer-bridge/beamer/pull/1560#discussion_r1147326658).

Ok, interesting. BTW, I completely understand where you are coming from and mathematically spoken I completely agree.
According to your link it seems to me that forex (foreign exchange) markets again have their own common lingo, which is again how ratios are interpreted mathematically. (Also it makes mathematically more sense because we end up with the unit we want to end in). 
However, as you also have pointed out, I think in the general financial context it is more common sense to interpret it as `ETH to token`.
Given the fact that it seems that even within the financial markets there is different interpretation I would suggest to go with the more explicit `--eth-to-token` naming.
Do you agree?

Reply:
> `--eth-to-token-rate` or just `eth-in-token`?

Whichever you prefer, I don't have a strong preference here.